tests/unit/browser/test_webelem.py

```diff
@@ -325,6 +327,38 @@ def test_debug_text(self, elem, xml, expected):
         elem._elem.toOuterXml.return_value = xml
         assert elem.debug_text() == expected
 
+    def test_remove_blank_target(self):
+        elem = get_webelem(tagname='a', attributes={'target': '_blank'})
+        elem.remove_blank_target()
+        assert elem._elem.attribute('target') == '_top'
```

Comment:
No need to access `_elem` here (and everywhere else you call `.attribute()`), you can simply do `assert elem['target'] == '_top'`.